types/react-dates/tslint.json

```diff
@@ -10,6 +10,7 @@
         "dt-header": false,
         "eofline": false,
         "export-just-namespace": false,
+        "file-name-casing": false,
```

Comment:
Uhhhh, why was this lint rule turned off? Keep this one on. For sure.

Reply:
If I enable this, tslint wants me to rename the file react-dates-tests.tsx to reactDatesTests.tsx. Doing so, a typescript test will fail, wanting me to rename the file from reactDatesTests.tsx to react-dates.tests.tsx. It's the only way I found to make every test pass.

Reply:
npm doesn't use the same casing convention tslint wants us to use, so we've disabled that rule globally.